src/doc/src/reference/config.md

```diff
@@ -321,7 +321,7 @@ all capital letters.
 
 This section documents all configuration keys. The description for keys with
 variable parts are annotated with angled brackets like `target.<triple>` where
-the `<triple>` part can be any target triple like
+the [`<triple>`] part can be any [target triple] like
```

Comment:
I feel like it's a bit too many links to the same source. Should we keep the latter only?

Reply:
You are absolutely right. Went a bit overzealous there.